Python/compile.c

```diff
@@ -3259,12 +3265,15 @@ static int
 compiler_break(struct compiler *c)
 {
     struct fblockinfo *loop = NULL;
+    struct compiler_unit origin_loc;
+    COPY_COMPILER_UNIT_LOC(c->u, &origin_loc);
```

Comment:
It would be a smaller change to just save the 4 ints, so for a backport I think that's better.

Reply:
Done!